docs/pages/web/link.js

```diff
@@ -449,9 +486,23 @@ Accessible content is critical if we consider that assistive technology also pre
           description={`
 When rendering an external link, add text for screen readers that this Link will go to a different destination. Users should be informed that they will be moving out of a domain and which domain they are moving to. This is particularly relevant for those with cognitive impairments or people relying on assistive technology.
 
-We recommend adding an external Link to inline content. However, use max of two external Links per paragraph when possible, as adding more than two icons in the same block of text can clutter the design and create readability issues.
+These are cases where external links should open in a new tab, as it supports Pinners to better navigate our product without disrupting their flow experience:
+
+  - When changing pages would make the user lose unsaved progress—for example, filling out a form or adding content.
+
+  - When starting a process, that would stop if the user clicks away. For example, watching a video or completing a core task.
+
+  - When Pinners are logged-in and, clicking away would interrupt that session.
+
+Sometimes opening links in a new tab could be challenging for users with difficulty perceiving visual content, cognitive disabilities, or people who magnify their screens. We don't recommend opening a link in a new tab when:
+
+  - It is invasive and disruptive to the flow of the page.
+
+  - Without clearly informing users about this behavior.
+
+We recommend adding an external link to inline content. However, use a maximum of two external links per paragraph when possible, as adding more than two icons in the same block of text can clutter the design and create readability issues.
 
-Also, consider adding external Links to elements where the "visit" icon will support the user's comprehension, letting them know they are leaving Pinterest content and going to an external domain. For example, links inside a footer element.
+Also, consider adding external links to elements where the "visit" icon will support the user's comprehension, letting them know they are leaving Pinterest content and going to an external domain. For example, links inside a footer element.
```

Comment:
grammar nit: I know it's not your change, but the last sentence is a fragment — there's no verb. Perhaps something like "For example, when using external links inside a footer element."